lib/cube_browser/__init__.py

```diff
@@ -93,3 +96,48 @@ def slider_coords(self):
                 available.append(self.cube.dim_coords[dim])
         return available
 
+
+class Browser(object):
+    """
+    Compiler for cube_browser plots and associated sliders.
+
+    Compiles a single cube_browser.Plot instance or list of instances into a
```

Comment:
i don't think we have a Plot class

please just put this as `cube_browser plot` in the docstring for now